.github/workflows/build.yml

```diff
@@ -41,6 +41,30 @@ jobs:
           cargo msrv --path derive-macros/ verify --all-features
           cargo msrv --path ffi/ verify --all-features
           cargo msrv --path ffi-proc-macros/ verify --all-features
+  msrv-run-tests:
```

Comment:
ideally i would say to just put this in `msrv` job above? but i guess we can't do that due to this using specific toolchain and not necessarily wanting that for the job above?

Reply:
yeah, i also want to parallelize this